### DISCUSSION TITLE
Entity permissions by web role view

### DISCUSSION THREAD

**Discussion by ZaarourOmar** [rappen/EntityPermissionVisualizer#4] [Ideas]
Hi Jonas,

Thinking about this from the other direction, If we can know what are the permissions for each web role, it will save us time because currently we have to go inside the entity permission record to see what web role is attached to it. 
So if we have another mode where the tool lists all the web roles, and when we click on one of them, it shows all all Entity permissions records for that web role along with the permissions themselves (read, write, etc). 

Another improvement on the above would be the ability to select multiple web role at once and see in some kind of simple visualization how they differ and how they intersect. In projects where there are many roles, this will be super useful to debug security issues. 

Great tool!

**Comment by rappen** [comment]
Sorry for the little "pause" from me...

Is the tool still usable?
Your feature still interesting?

Don't want to work on for no one... 😉
